operator/data/profiles/default.yaml

```diff
@@ -662,33 +662,41 @@ spec:
       coreDNSPluginImage: istio/coredns-plugin:0.2-istio-1.1
 
     kiali:
-      hub: quay.io/kiali
-      tag: v1.14
-      contextPath: /kiali
+      operator:
```

Comment:
I would like other's opinions if telemetry should be in the default profile. I think it would be great if Istio's operator code could handle telemetry installation (using CRs) independently of istiod installation.

If the anwer is no to the aboe questions, then a different profile - perhaps "full" or something of the like would make more sense.